dotCMS/src/main/java/com/dotmarketing/image/filter/CropImageFilter.java

```diff
@@ -25,13 +29,18 @@
 	public File runFilter(File file,  Map<String, String[]> parameters) {
 		int x = parameters.get(getPrefix() + "x") != null ? Integer.parseInt(parameters.get(getPrefix() + "x")[0]) : 0;
 		int y = parameters.get(getPrefix() + "y") != null ? Integer.parseInt(parameters.get(getPrefix() + "y")[0]) : 0;
-		int w = parameters.get(getPrefix() + "w") != null ? Integer.parseInt(parameters.get(getPrefix() + "w")[0]) : 0;
-		int h = parameters.get(getPrefix() + "h") != null ? Integer.parseInt(parameters.get(getPrefix() + "h")[0]) : 0;
-		if (w == 0 || h == 0) {
-			return file;
-		}
+		final float wIn = parameters.get(getPrefix() + "w") != null ? Float.parseFloat(parameters.get(getPrefix() + "w")[0]) : 0f;
+		final float hIn = parameters.get(getPrefix() + "h") != null ? Float.parseFloat(parameters.get(getPrefix() + "h")[0]) : 0f;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid declaring a variable if it is unreferenced before a possible exit point.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4761083)